docs/hub/datasets-adding.md

```diff
@@ -6,8 +6,8 @@ Any Hugging Face user can create a dataset! You can start by [creating your data
 * [Push files with the `push_to_hub` method from 🤗 Datasets](https://huggingface.co/docs/datasets/upload_dataset#upload-from-python)
 * [Use Git to commit and push your dataset files](https://huggingface.co/docs/datasets/share#clone-the-repository)
 
-While it's possible to add raw data to your dataset repo in a number of formats (JSON, CSV, Parquet, text, and images), for large datasets you may want to [create a loading script](https://huggingface.co/docs/datasets/dataset_script#create-a-dataset-loading-script). This script defines the different configurations and splits of your dataset, as well as how to download and process the data.
+While in many cases it's possible to just add raw data to your dataset repo in any supported formats (JSON, CSV, Parquet, text, images, audio files, …), for some large datasets you may want to [create a loading script](https://huggingface.co/docs/datasets/dataset_script#create-a-dataset-loading-script). This script defines the different configurations and splits of your dataset, as well as how to download and process the data.
 
 ## Datasets outside a namespace
 
-Datasets outside a namespace are maintained by the Hugging Face team on GitHub. Unlike the naming convention used for community datasets (`username/dataset_name` or `org/dataset_name`), datasets outside a namespace can be referenced directly by their name (e.g. [`glue`](https://huggingface.co/datasets/glue)). If you find that an improvement is needed, refer to the [🤗 Datasets documentation](https://huggingface.co/docs/datasets/main/en/share#datasets-on-github-legacy) for an explanation on how to submit a PR on GitHub to propose edits.
\ No newline at end of file
+Datasets outside a namespace are maintained by the Hugging Face team on GitHub. Unlike the naming convention used for community datasets (`username/dataset_name` or `org/dataset_name`), datasets outside a namespace can be referenced directly by their name (e.g. [`glue`](https://huggingface.co/datasets/glue)). If you find that an improvement is needed, refer to the [🤗 Datasets documentation](https://huggingface.co/docs/datasets/main/en/share#datasets-on-github-legacy) for an explanation on how to submit a PR on the Hub to propose edits.
```

Comment:
I think we should update the datasets docs about [Datasets on GitHub (legacy)](https://huggingface.co/docs/datasets/main/en/share#datasets-on-github-legacy):
- since completely removed from GitHub, they are maintained on the Hub (as the rest of datasets)
- contributors should use the corresponding community tab to inform about issues and/or open PRs
```suggestion
Datasets outside a namespace are maintained by the Hugging Face team. Unlike the naming convention used for community datasets (`username/dataset_name` or `org/dataset_name`), datasets outside a namespace can be referenced directly by their name (e.g. [`glue`](https://huggingface.co/datasets/glue)). If you find that an improvement is needed, use their "Community" tab to open a discussion or submit a PR on the Hub to propose edits.
```
The rephrasing could be otherwise...